pkgs/desktops/deepin/go-package/dde-daemon/0001-fix-wrapped-name-for-verifyExe.patch

```diff
@@ -0,0 +1,30 @@
+From 159ffabd16519c9c21155b367d98e0d09593e738 Mon Sep 17 00:00:00 2001
+From: rewine <luhongxu@deepin.org>
+Date: Tue, 31 Jan 2023 17:48:06 +0800
+Subject: [PATCH] fix wrapped name for verifyExe
+
+---
+ dock/process_info.go | 7 +++++++
+ 1 file changed, 7 insertions(+)
+
+diff --git a/dock/process_info.go b/dock/process_info.go
+index 83c61d58..cab8f7a5 100644
+--- a/dock/process_info.go
++++ b/dock/process_info.go
+@@ -119,6 +119,13 @@ func verifyExe(exe, cwd, firstArg string) bool {
+ 		return false
+ 	}
+ 	logger.Debugf("firstArgPath: %q", firstArgPath)
++	if exe == firstArgPath {
++		return true
++	}
++	if strings.HasSuffix(exe, "wrapped") {
++		exeBase := filepath.Base(exe)
++		exe = exe[0:len(exe)-len(exeBase)] + exeBase[1:len(exeBase)-8]
++	}
```

Comment:
So what does `verifyExe` actually do? Can we just return true whatever.

Reply:
Verify that the file being run (exe) is consistent with the startup command (cmd)
Failure to pass the verification will result in failure to update `ProcessInfo`
Judging the situation of wrapped exe should be better than return true directly